### PR TITLE
Renamed schema variable name to schema_ as there is a schema function defined

### PR DESCRIPTION
**Pull request by ShahriyarR**
## Change Summary

Spotted an anti-pattern which can lead to shadowing in the outer scope, as there is a function named `schema`.
Renamed all `schema`  variables to  `schema_`

## Checklist

* [ ] Unit tests for the changes exist
* [ ] Tests pass on CI and coverage remains at 100%
* [ ] Documentation reflects the changes where applicable
* [x] `changes/<pull request or issue id>-<github username>.md` file added describing change
  (see [changes/README.md](https://github.com/samuelcolvin/pydantic/blob/master/changes/README.md) for details)

### CODE REVIEW

**Comment by samuelcolvin** [comment]
This looks good. Please could you add a change description.

Also, in future please use the PR template.

**Comment by ShahriyarR** [comment]
I have already sent one without the template usage, sorry for that.
I am going to use it there also.

File: changes/2724-shahriyarr.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Spotted an anti-pattern which can lead to shadowing in the outer scope, as there is a function named schema.
+Renamed all schema variables to schema_
```
Comment (samuelcolvin) [inline changes/2724-shahriyarr.md:2]:
```suggestion
Renamed variable `schema` to `schema_` to avoid shadowing of global variable name.
```

**Review by PrettyWood — APPROVED** [review]

**Comment by PrettyWood** [comment]
Thanks! 🙏

**Pull request merged**